### DISCUSSION TITLE
Is there any way can disable clangd's snippets completion?

### DISCUSSION THREAD

**Discussion by NEX-S** [clangd/clangd#1296] [Q&A]
I just wanna using my own snippets :D

**Comment by tom-anders** [comment]
Setting "snippetSupport"=false in the client capabilities should do the trick (see https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_completion). The exact way how to do this depends on your editor. E.g. for neovim's native LSP: https://github.com/neovim/nvim-lspconfig/wiki/Snippets

**Reply by NEX-S** [reply]
ohh, I'm using the sublime text, and this is my sublime's lsp config
```json
    "clients": {
        "clangd": {
            "enabled": true,
            "command": [
                "clangd"
            ],
            "selector": "source.c | source.c++ | source.objc | source.objc++",
            "auto_complete_selector": "punctuation.accessor | (meta.preprocessor.include string - punctuation.definition.string.end)",
            "disabled_capabilities": {
                "hoverProvider": false,
                "snippetsSupport": false,
            }
        },
    },
```
but seems it don't work :(

**Reply by ghost** [reply]
> Setting "snippetSupport"=false in the client capabilities should do the trick (see https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#textDocument_completion). The exact way how to do this depends on your editor. E.g. for neovim's native LSP: https://github.com/neovim/nvim-lspconfig/wiki/Snippets

Hey, I'm on the same problem, had overridden "textDocument.completion.completionItem.snippetSupport" to false, tried to true as well, it just not working, could you tell me is it working on your side? I think I properly configured my editor, but clangd doesn't read this preference
